tensorflow/lite/kernels/activations.cc

```diff
@@ -112,6 +117,33 @@ TfLiteStatus GenericPrepare(TfLiteContext* context, TfLiteNode* node) {
                                TfLiteIntArrayCopy(input->dims));
 }
 
+void* LeakyReluInit(TfLiteContext* context, const char* buffer, size_t length) {
+  return new LeakyReluOpData;
+}
+
+void LeakyReluFree(TfLiteContext* context, void* buffer) {
+  delete reinterpret_cast<LeakyReluOpData*>(buffer);
+}
+
+TfLiteStatus LeakyReluPrepare(TfLiteContext* context, TfLiteNode* node) {
+  TF_LITE_ENSURE_EQ(context, NumInputs(node), 1);
+  TF_LITE_ENSURE_EQ(context, NumOutputs(node), 1);
+  const TfLiteTensor* input = GetInput(context, node, 0);
+  TfLiteTensor* output = GetOutput(context, node, 0);
+  TF_LITE_ENSURE_EQ(context, input->type, output->type);
+
+  LeakyReluOpData* data = reinterpret_cast<LeakyReluOpData*>(node->user_data);
+
+  if (output->type == kTfLiteUInt8) {
+    double real_multiplier =
+        input->params.scale * input->params.scale / output->params.scale;
```

Comment:
Sorry but I am a bit confused about this logic, why are there two input->params.scale?

Also, I think you can merge the logic of q_alpha here as well so in the kernel you can change
params.output_offset + MultiplyByQuantizedMultiplierSmallerThanOneExp(
                                     input_value * alpha_value,
                                     params.output_multiplier,
                                     params.output_shift)
to 
params.output_offset + MultiplyByQuantizedMultiplierSmallerThanOneExp(
                                     input_value,
                                     params.output_multiplier,
                                     params.output_shift)

Reply:
@jianlijianli thanks for the review, the logic of two input->params.scale is in sync with the formula as i have quantized the alpha using the same scale as of input, so real multipler will be 
((input A scale * input B scale) / output scale). I hope this is ok.

About your last part of the comment i could not understand. Can you be more specific as i have to multiply the updated alpha value to input for Leaky relu to produce a correct output.

Regdrs
Amit